pkgs/development/libraries/curlcpp/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, cmake, curl }: 
+
+stdenv.mkDerivation rec {
+  name = "curlcpp";
```

Comment:
The version must be part of the `name` (the version is actually parsed out of the `name` attribute, the `version` attribute has no special significance).